src/denish/Assignment63.java

```diff
@@ -0,0 +1,31 @@
+package denish;
+
+public class Assignment63 {
+
+	boolean primeNumber(int num) {
+		for (int i = 2; i <= num / 2; i++) {
+			if (num % i == 0) {
+				return false;
+			}
+		}
+		return true;
+	}
+
+	void sumOfPrimeNum() {
+		int count = 0;
+		int sum = 0;
+		for (int num = 50; num <= 150; num++) {
```

Comment:
You have to pass 2 numbers as parameter, here 50 and 150 is just for reference, you have to make your code dynamic